main.tf

```
@@ -50,6 +50,15 @@ resource "azurerm_linux_function_app" "this" {
 
   virtual_network_subnet_id = var.virtual_network_subnet_id
 
+  dynamic "sticky_settings" {
+    for_each = length(var.sticky_settings_app_setting_names) > 0 || length(var.sticky_settings_connection_string_names) > 0 ? [""] : []
```

Which value we put here isn't important, what's important is that there's a **single** value in this list, however we usually put `0` so I suggest we change to that to stay consistent.

```suggestion
    for_each = length(var.sticky_settings_app_setting_names) > 0 || length(var.sticky_settings_connection_string_names) > 0 ? [0] : []
```